test/Serialization/upgrades/runtests.jl

```diff
@@ -59,4 +59,8 @@
       "Vector" => 49:97,
     ])
   end
+
+  @testset "< 1.5.0 Upgrade" begin
```

Comment:
```suggestion
  @testset "< 1.6.0 Upgrade" begin
```
Since these serializations are only available from Oscar 1.5.0, the earliest upgrade that could happen is the 1.6.0 one.
But I agree that this seems somehow confusing, so I would suggest to rename all of these testsets in the following manner:

```suggestion
  @testset "load files serialized with 1.5.0 " begin
```

Reply:
I think what's also confusing is the other testset that says less than 1.4.0 and uses the version_1_3_0 folder. To be consistent I think we should change the name of the testset there too.

Reply:
> I think what's also confusing is the other testset that says less than 1.4.0 and uses the version_1_3_0 folder. To be consistent I think we should change the name of the testset there too.

Yeah, that's what I meant. All of the `< x.y.z Upgrade` test sets should be renamed e.g. like I suggested